lighthouse-core/report/html/report-styles.css

```diff
@@ -1341,3 +1341,9 @@
6FObWLUevaBR8kzjJI1VnvPOPI/D2nLz0Yve/b2P3P5ul9vdNuaIsu+7ebbddO0xvt/NlNR1XeZlJ/MpcmtkdSdDxSmDLjLOdCDf1wzAY9eGPBPBchwE2ym2RjOvccWsTVdV4NioUCYbja+VMLpXEaUWVrZ5x1sb1F8/KB8V2eRiotCsSOGH3idKh91qWEROe8ZIlLYdsOOycSeMmjLOOKbTqtR2KINrFdmLO7mR+hOiE7caW72ZPVNKxq2pRMm14uoLn4sBSGIBygLwdMgaaBX3bpTGEhI+txjDnr4Vw6wxHnQcXD/bOd+ZY6CEAiXyCmBF/gWalBS3cuXS16XtmwPnG/uhGvty4Vtd2gs1Y/ao9MSvlcvmootiCc9F4dC1EOsjZEAd7cwalTJ+O6lJB6dh9JZnq/UWO7Mxq9XL6GarXHGfOEsNDbWqoSgkjPOFc4ZLy9DrSjlaG0bF5QSuyHnihAsnZcWbJfrATGVKPAvbrLtxbjiIkVmeeWEco11aFVHKTiRpwY04sjuiBsgEMtj5iUsGexY/HaWCUtz1q2EYhe3j9SHGdFcLyfbtpaeB7SZwEi/u4Jvf+cKZuiYnFqDKiY5BZOn66vrQn+W7CKdKt1/Jb/UDt49bq/Ym4Blwxgt8o6g1Bw1uJdrURRy1sMJtNkw+04e1KR2D/gkXYyGfFqz5ZhfvOX2AvuKroARq78BN112XuCw9GwxkxsjQiXTaouxXJyR8Z8Ep2jWF0sv2XPkFZgnsNQd8xuIUPEXaqXO50ODAajKQIll6YOk53yEbc0wDlrQUOgVpIrkUpxlyZBSlgOMAItPUZ/tSMDQUl+SBYXx4VmdI4rsPU6PYq4mc4a8sC9+jR/nooNOYVGRxWROG1YHpEe/6jg4A/HfT37Wl/m0J1ijZtmaIYjldJek5GDtPdbM7BlYw7c0riWKeRZLB4yTkeH7lWkDwkqRTZRY6qT0iolGUvKbB149mx1GFqs3ln9gCjBMl/81nDK5WeMhPqbIZCrt0w4+3cWlZDDhe7LyiMN6ud/ZPP5A6AnSPW2exqSLYkKQyiZYuVKp12fCTx3AiVlGbaHflgnXPsiaUdjLQ+tlvu5hYG3UcYkqx6l5AM7cOJGiaol3wqgLgMqXf/fcTqBeq6XRAsJKGUaxCOy1QqlKzXJUZIVVGvHLK1zaVW4GKPYSQg9rU6EZWTpGJRrmlmksShUngSsVYRC0A/tXY8hYZOdx4Sn51YJUBGSQVa7Gi2JWmLIPz3UG1vfViyc+y7/0mFwpLoXtrbZmZ2vJXi2fO//pDYuj72OJ+odLFI07LZlCTziUKHKXZHg2iKWJUE32aNV2qvlm2oqNa85UgNtw579Um7Sv0SIxmL6JgybouKrpfLJoeRk2nXKIRpdHJUAohbsiXLaxyOBoezSXjerXYEBkSWw4YL9b+KLvemLNyFYiGlg5zqED+Xyb1aQ4lDuqMoWGw06TDdy8aSGR4J/7YnvgWTLdp2VmohJWEL0h0W5Zp1Zpxk6UgIU5w0z7UgSacmn6BqnVeXCEIDDxkBE648n7JfW9WR029zhuugIw6fdeQKgqh8CEZB6VLmrk6l1F7V82OyYy/TurCdzFL7MFlt/nMHH9XZ/bAYIv93JXwVWEtGHbadkqB6j7673GYkgzw0Rggqb+rLguzLjw8izENxE1a1gq70ewTEi2XrxOzUlSrWWSYyPfrwQUTANY0hiAuw/pDxTHWWrwNBDc6LKWNdD/Wo2uUPLkwx55W7209MTWHKot4g7vSb+klZr0YOPPCXswYjC6J6LiDIpMoGv+tQdrbN8Gy//MJ9n2U51GkRIHJn0VL6Dn9AwXQhfnGVZaNLZij+x2wVhbvVZJBvRDHXCbUy5tDWGqDIdZaOvDvLgQ4zD9/4lB1h0rxEdcZYcun/3EPgr2qjOncvxJM3mgPvf+yDJM2GypIEq9PhLNnUkU5460na6pTqXdNqrT9sB0TWp/RoBHykQW8yi/vNFjDzLtw8aMfnH9+rLDCoeO2PXKDqc2PKPrSDqXuAQwFChPeStoty10tBMSGssqUHsci97IrZU0+SzWXfWuqC/GkOBG7/N4XGcCCvOo64j7u9LANvdakHOO6inIez9cWPNi2+Xr+uNMJCeoznl0XEfTqrtqSSyiJBXNKyDJUZX3h3VqyYuHOes0xCVXEs7+fAemsWfwkwUNnFtrA2PdvFNrUmgxecpCN26DbC0f7dYydnZvNDaU3B9EpSVE37AkxV5AycmpycHTreY8k7ExxlgRzXeWJ8pl67pT2dkGodKZEnV4vt4rvD7kVlXS3eHqKPZ1lWQXjFeHY3kkO2HJXKrNZNk30GS5MO4rN6i6RtCyfLtiKaTlew4K9ius3CF1eHlWK3hFhuabeULqcp4qi9RZUipO1R8I/dd3JLdjEjMSvo5B8m6RX84efsbHYhfoaVdH+pUK7fuSeU/qi180vY6jqUJ/PAPNJi2Hkh4KMGN2XOeR5Y5Zf99TPB7eLLxmgL3f1YembDnMzsLF9QEszvZC6AqYO+Wemk9HW0uk2USVnqwqBrqXA83wR3zW+uUpBMawQtEQTWvJjcmLU6HVr4uN9BWE+VZTrneuWsqqbA+JQeVIyEaa2ZsU5WGlpB0sg4W03SJNbipw2ajzeiGk9SBL5+rLQiiTyI3jAZ8D1Mxfo4xE7v86JuQuzkGUt28mGH5FPFsAxT5yUFh+wdCt5U16/Q6mqhblTLhkZl6mBK2xj/QpXri74MqugJXOXpcL6UNKiULOWD/r5gb0Ed2INCcP6jAmxFQKwJYckJyi0c4U9+izkVyjLH8ab9SivliNcoxR3JitIni/vh3cp6eVpW/mie3YwkZizxR/CdGryx1JfRjK9WB0bwKsBKqBkvPs4y/s4Deb199ma7VZGLvoMtdg4mGqFNPT9ct4Akg+0zRvy7o8w0QAc1cYyEoHxUD8UvHEEQH3txuO4KFiEoiNcYJeRtmWilqRRSQGSEcdwkrExdN/1
```

Comment:
RIP scrollbar :(

Reply:
another candidate for an enriched asset pipeline (like #8632)

Reply:
yeah +1 @Hoten's approach is suddenly much more necessary :)